.github/workflows/frontend-qa.yml

```
@@ -41,3 +65,17 @@ jobs:
 
       - name: Run E2E smoke tests
         run: pnpm test:e2e
+
+"scripts": {
```

actionlint (1.7.12)

[error] unexpected key "scripts" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)